platforms/core-runtime/internal-instrumentation-processor/src/test/groovy/org/gradle/internal/instrumentation/extensions/property/PropertyUpgradeCodeGenTest.groovy

```diff
@@ -361,7 +361,7 @@ class PropertyUpgradeCodeGenTest extends InstrumentationCodeGenTest {
     private static String getDefaultDeprecation(String className, String propertyName) {
         return "DeprecationLogger.deprecateProperty(" + className + ".class, \"" + propertyName + "\")\n" +
             ".withContext(\"Property was automatically upgraded to the lazy version.\")\n" +
```

Comment:
❓ I wonder if this could be phrased better. It's not the property that was upgraded, but the... invocation to the old API addressing the eager property? Maybe we should ask Laura for help here? But this seems to be important to get right, as the whole magical nature of the process could get users confused easily.

Reply:
@lkasso Any opinion here how to express this better? Here we want to say, that invocation that used eager property was replaced with a new invocation that is backed by a lazy property.